File: specs/master/cluster/nodes_info/node_jvm_info.ts

/**namespace:Cluster.NodesInfo */
interface node_jvm_info {
	pid: integer;
	version: string;
	vm_name: string;
	vm_version: string;
	vm_vendor: string;
	memory_pools: string[];
	gc_collectors: string[];
	start_time_in_millis: long;
	mem: node_info_j_v_m_memory;
}

File: output/csharp/Cluster/NodesInfo/NodeJvmInfo.cs
using Nest.Cluster;
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cluster {

	public class NodeJvmInfo  {
		
		[DataMember(Name="gc_collectors")]
		public List<string> GcCollectors { get; set; }


		[DataMember(Name="mem")]
		public NodeInfoJvmMemory Mem { get; set; }


		[DataMember(Name="memory_pools")]
		public List<string> MemoryPools { get; set; }


		[DataMember(Name="pid")]
		public int Pid { get; set; }


		[DataMember(Name="start_time_in_millis")]
		public long StartTimeInMillis { get; set; }


		[DataMember(Name="version")]
		public string Version { get; set; }


		[DataMember(Name="vm_name")]
		public string VmName { get; set; }


		[DataMember(Name="vm_vendor")]
		public string VmVendor { get; set; }


		[DataMember(Name="vm_version")]
		public string VmVersion { get; set; }

	}
}
